charts/etcd/templates/etcd-copy-job.yaml

```diff
@@ -0,0 +1,272 @@
+{{- if  .Values.sourceStore}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.copyJobName }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    gardener.cloud/owned-by: "{{ .Release.Namespace }}/{{ .Values.name }}"
+    gardener.cloud/owner-type: "etcd"
+{{- if .Values.annotations }}
+{{ toYaml .Values.annotations | indent 4 }}
+{{- end }}
+  labels:
+    networking.gardener.cloud/to-public-networks: allowed
+    name: etcd
+    instance: {{ .Values.name }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+  ownerReferences:
+  - apiVersion: druid.gardener.cloud/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: Etcd
+    name: {{ .Values.name }}
+    uid: {{ .Values.uid }}
+spec:
+  template:
+    metadata:
+      labels:
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: etcd-copy
+        image: {{ .Values.backup.image }}
+        imagePullPolicy: Always
+        command:
+        - etcdbrctl
+        - copy
+        - --snapstore-temp-directory=/var/etcd/data/tmp
+{{- if  .Values.store.storageProvider }}
+        - --storage-provider={{ .Values.store.storageProvider }}
+{{- end }}
+{{- if .Values.store.storePrefix }}
+        - --store-prefix={{ .Values.store.storePrefix }}
+{{- end }}
+{{- if .Values.store.storageContainer }}
+        - --store-container={{ .Values.store.storageContainer }}
+{{- end }}
+{{- if  .Values.sourceStore.storageProvider }}
+        - --source-storage-provider={{ .Values.sourceStore.storageProvider }}
+{{- end }}
+{{- if .Values.sourceStore.storePrefix }}
+        - --source-store-prefix={{ .Values.sourceStore.storePrefix }}
+{{- end }}
+{{- if .Values.sourceStore.storageContainer }}
+        - --source-store-container={{ .Values.sourceStore.storageContainer }}
+{{- end }}
+{{- if .Values.copySpec.maxAge }}
+        - --max-age-days={{ .Values.copySpec.maxAge }}
+{{- end }}
+{{- if .Values.sourceStore.snapshotsToCopy }}
+        - --last-n-snapshots={{ .Values.copySpec.snapshotsToCopy }}
+{{- end }}
```

Comment:
Let's adapt the naming of values and command line parameters to whatever we agree on in `etcd-backup-restore` (see also above).